src/Raven.Server/Documents/Handlers/Processors/Debugging/StorageHandlerProcessorForGetScratchBufferReport.cs

```diff
@@ -28,12 +28,19 @@ protected override async ValueTask HandleCurrentNodeAsync()
             var name = GetName();
             var typeAsString = RequestHandler.GetStringQueryString("type", false);
 
+            var envs = RequestHandler.Database.GetAllStoragesEnvironment();
+
+            if (envs == null)
```

Comment:
in what case we can have a null here? Please explain.

Reply:
We can't. only when we look for specific DB it can